app/src/main/java/io/github/gsantner/memetastic/activity/MainActivity.java

```diff
@@ -143,20 +156,19 @@ protected void onCreate(Bundle savedInstanceState) {
         _drawer.addDrawerListener(toggle);
         toggle.syncState();
         _navigationView.setNavigationItemSelectedListener(this);
-        _tabLayout.setOnTabSelectedListener(this);
+        //_tabLayout.setOnTabSelectedListener(this);
 
         _tagKeys = getResources().getStringArray(R.array.meme_tags__keys);
         _tagValues = getResources().getStringArray(R.array.meme_tags__titles);
 
 
-        // Setup Floating Action Button
-
-
+        // Setup Floating Action
         _recyclerMemeList.setHasFixedSize(true);
         _recyclerMemeList.setItemViewCacheSize(app.settings.getGridColumnCountPortrait() * app.settings.getGridColumnCountLandscape() * 2);
         _recyclerMemeList.setDrawingCacheEnabled(true);
         _recyclerMemeList.setDrawingCacheQuality(View.DRAWING_CACHE_QUALITY_LOW);
         _recyclerMemeList.addItemDecoration(new GridDecoration(1.7f));
+
```

Comment:
do we neer any RecylcerView/adapter in the MainActivity itself? Its all m in MemeFragment now right that is needed or?

Reply:
yes, bcoz this recycler holds data for the fav and saved mode, it is replaced by view pager for create mode.

Reply:
yes, but couldn't both be replaced by a MemeFragment, instead of having them directly in MainActivity? So to just use MainActivity with placeholder, and inserting/swapping MemeFragment's there

Reply:
but this can also be done in another PR if you want to do that

Reply:
ok I will see to it.